terraform/environments/data-platform/lambda.tf

```diff
@@ -437,11 +445,11 @@ module "data_product_push_to_catalogue_lambda" {
   tags                           = local.tags
   description                    = "Pushes metadata to openmetadata catalogue"
   role_name                      = "push_to_catalogue_role_${local.environment}"
-  policy_json                    = data.aws_iam_policy_document.read_openmetadata_secrets.json
+  policy_json                    = data.aws_iam_policy_document.iam_policy_document_for_push_to_catalogue_lambda.json
   policy_json_attached           = true
   function_name                  = "data_product_push_to_catalogue_${local.environment}"
   create_role                    = true
-  reserved_concurrent_executions = 1
+  reserved_concurrent_executions = 100
```

Comment:
Do we want to limit this a bit further? Not sure how many incoming requests the catalogue can handle without falling over